### PR TITLE
fix: Removes inline image if it is the first item in the article content

### PR DESCRIPTION
**Pull request by Geoff-Ford**
https://nidigitalsolutions.jira.com/browse/TNLT-4684

**Before:**
Screenshot 2020-09-11 at 18 48 19

**After:**
Screenshot 2020-09-11 at 18 49 42

PS: Added `Dimensions` to our `modulePaths` as it was missing.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -85,7 +85,7 @@
     lodash "^4.17.13"
     source-map "^0.5.0"
 
-"@babel/generator@^7.11.0", "@babel/generator@^7.5.0":
+"@babel/generator@^7.11.0":
```
Comment (c-j-j) [inline yarn.lock:88]: Just double checking - did you mean to push an updated yarn.lock?

**Review by c-j-j — APPROVED** [review]

File: lib/android/modulePaths.js [hunk 2]
```diff
@@ -93,6 +93,7 @@ module.exports = [
   "node_modules/react-native/Libraries/Performance/Systrace.js",
   "node_modules/react-native/Libraries/StyleSheet/flattenStyle.js",
   "node_modules/react-native/Libraries/TurboModule/TurboModuleRegistry.js",
+  "node_modules/react-native/Libraries/Utilities/Dimensions.js",
```
Comment (aronshamash) [inline lib/android/modulePaths.js:96]: I guess this shouldn't be here.

**Review by aronshamash — CHANGES_REQUESTED** [review]

File: lib/android/modulePaths.js (hunk 2 above)
Comment (Geoff-Ford) [reply lib/android/modulePaths.js:96]: Yep, it should. It was missing before.

File: yarn.lock (hunk 1 above)
Comment (Geoff-Ford) [reply yarn.lock]: Updated yarn to 1.22.5, double checked on right node version (10.13.0) via nvm, deleted and regenerated `yarn.lock`

**Review by aronshamash — APPROVED** [review]

**Pull request merged**